extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -123,13 +128,13 @@ export class DataTierApplicationWizard {
 	}
 
 	public setPages(): void {
-		let selectOperationWizardPage = azdata.window.createWizardPage(loc.selectOperationPageName);
-		let deployConfigWizardPage = azdata.window.createWizardPage(loc.deployConfigPageName);
-		let deployPlanWizardPage = azdata.window.createWizardPage(loc.deployPlanPageName);
-		let summaryWizardPage = azdata.window.createWizardPage(loc.summaryPageName);
-		let extractConfigWizardPage = azdata.window.createWizardPage(loc.extractConfigPageName);
-		let importConfigWizardPage = azdata.window.createWizardPage(loc.importConfigPageName);
-		let exportConfigWizardPage = azdata.window.createWizardPage(loc.exportConfigPageName);
+		let selectOperationWizardPage = azdata.window.createWizardPage(loc.selectOperationPageName, loc.selectOperationPageName);
```

Comment:
The telemetry names shouldn't be localized - doing that makes it difficult to correlate all the events for a particular page across different language installs.